index-handlers/src/test/java/no/sikt/nva/nvi/index/IndexDocumentHandlerTest.java

```diff
@@ -752,13 +752,13 @@ private Candidate randomApplicableCandidate() {
 
     private Candidate randomApplicableCandidate(URI topLevelOrg, URI affiliation) {
         var request = createUpsertCandidateRequest(topLevelOrg, affiliation);
-        Candidate.upsert(request, candidateRepository);
+        Candidate.upsert(request, candidateRepository, periodRepository);
         return Candidate.fetchByPublicationId(request::publicationId, candidateRepository, periodRepository);
     }
 
     private Candidate randomApplicableCandidate(URI topLevelOrg, URI affiliation, ChannelType channelType) {
```

Comment:
Linter complaint: `Value of parameter 'topLevelOrg' is always 'IndexDocumentTestUtils. HARD_CODED_TOP_LEVEL_ORG' `